dashboard/lib/policies/child_account.rb

```diff
@@ -44,7 +44,6 @@ module ComplianceState
   # parent permission before the student can start using their account.
   def self.compliant?(user)
     return true unless parent_permission_required?(user)
-    return true if user_predates_policy?(user)
```

Comment:
Because of this, "existing" users are always CAP compliant, so they will never be locked out.